local/Readme.md

```diff
@@ -0,0 +1,34 @@
+# Setup NextCloud
+
```

Comment:
This file is in pt_BR then must go to the doc pt_BR

Reply:
do you think there should have an english version too?

Reply:
Yes, the README is in 2 languages.